soc/raspberrypi/soc.yml

```diff
@@ -4,3 +4,4 @@ family:
   - name: rp2xxx
```

Comment:
this would need to be split into the 2 series, with matching names

Reply:
OK, thanks. Fixed.